subworkflows/nf-core/merge_index_bam.nf

```diff
@@ -15,19 +15,27 @@ workflow MERGE_INDEX_BAM {
     ch_versions = Channel.empty()
 
     // Figuring out if there is one or more bam(s) from the same sample
-    bam.branch{
-        single:   it[1].size() == 1
-        multiple: it[1].size() > 1
+    bam.groupTuple().branch{ meta, bam ->
+        single:   bam.size() == 1
+            [meta, bam.transpose()[0]]
```

Comment:
This looks weird. What's going on here?
```suggestion
            [meta, *bam]
```
Is this what you want?

Reply:
not sure actually, just want to get the bam file(s) out of their `[]` due to `groupeTuple()`

Reply:
but never had to do any of that before, so might be something else happening here, I'll dug that part more

Reply:
Putting a * in front of a list takes off the outer brackets ( spread operator ), but yes, I agree. Look at why groupTuple is making a list of list.

Reply:
actually, it's coming from a previous subworkflow, so digging into that now